app/controllers/TaxYearsController.scala

```diff
@@ -48,9 +48,10 @@ class TaxYearsController @Inject()(taxYearsView: TaxYears,
                     isAgent: Boolean,
                     origin: Option[String] = None)
                    (implicit user: MtdItUser[_]): Future[Result] = {
+    val maxTaxYears = 5
     Try {
       taxYearsView(
-        taxYears = user.incomeSources.orderedTaxYearsByAccountingPeriods.reverse,
+        taxYears = user.incomeSources.orderedTaxYearsByAccountingPeriods.reverse.take(maxTaxYears),
```

Comment:
I don't think we should move maxTaxYears into config, but worth define a separate private val as we have on line 46 in this controller =>
`private val earliestSubmissionTaxYear = 2023`
and maybe create a separate private method to extract this range of tax years